src/models/Like.js

```diff
@@ -0,0 +1,54 @@
+
+export default (sequelize, DataTypes) => {
+  const Like = sequelize.define('Like', {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true
+    }
+  });
+
+  // eslint-disable-next-line func-names
+  Like.associate = function ({ User }) {
```

Comment:
Woof! Use ES6 arrow functions.

Reply:
Please check replies to previous reviews